### PR TITLE
Add epochs module

### PR DESCRIPTION
**Pull request by hieuvubk**
https://github.com/cosmos/cosmos-sdk/pull/19697

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -71,6 +71,7 @@ <h2>cosmossdk.io/</h2>
         <li><a href="/x/nft">cosmossdk.io/x/nft</a></li>
         <li><a href="/x/tx">cosmossdk.io/x/tx</a></li>
         <li><a href="/x/upgrade">cosmossdk.io/x/upgrade</a></li>
+        <li><a href="/x/epochs">cosmossdk.io/x/upgrade</a></li>
```
Comment (julienrbrt) [inline index.html]: Can you delete this? We try to have on the index only released stuff

**Review by julienrbrt — APPROVED** [review]

**Review by likhita-809 — COMMENTED** [review]
I think there should be `vangen.json` file changes with addition of `x/epochs` to it
cc: @julienrbrt

**Comment by hieuvubk** [comment]
> I think there should be `vangen.json` file changes with addition of `x/epochs` to it cc: @julienrbrt

Thanks, added

**Review by likhita-809 — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -71,6 +71,7 @@ <h2>cosmossdk.io/</h2>
         <li><a href="/x/nft">cosmossdk.io/x/nft</a></li>
         <li><a href="/x/tx">cosmossdk.io/x/tx</a></li>
         <li><a href="/x/upgrade">cosmossdk.io/x/upgrade</a></li>
+        <li><a href="/x/epochs">cosmossdk.io/x/epochs</a></li>
```
Comment (julienrbrt) [inline index.html]:
```suggestion
```

**Pull request merged**